mandatory/1-syntax-errors.js

```diff
@@ -1,16 +1,17 @@
 // There are syntax errors in this code - can you fix it to pass the tests?
 
-function addNumbers(a b c) {
+function addNumbers(a, b, c) {
   return a + b + c;
 }
```

Comment:
Comment: Well done! Same as answers as me. 
Consider: It makes no difference in the outcome. Sometimes you have space in between curly brackets and sometimes not. I do the same.

Reply:
Good work Saliha. 👍